lib/utensils/database_cleaner.rb

```diff
@@ -1,6 +1,14 @@
 RSpec.configure do |config|
+  config.before(:suite) do
+    DatabaseCleaner[:mongoid].strategy = :truncation
+    DatabaseCleaner[:active_record].strategy = :transaction
```

Comment:
simplified in 3530d4b

The `DatabaseCleaner.clean_with :truncation` is actually performing a TRUNCATE before the test suite, I found myself sometimes have some data ended up not being cleaned but can't find a reliable way to reproduce.. so dropping this option for now unless someone else also encounters this problem :grin: 

```sql
TRUNCATE TABLE "users","taggings", "posts", "other tables.." RESTART IDENTITY CASCADE;
```